fdbclient/Status.cpp

```diff
@@ -0,0 +1,9 @@
+#include "flow/flow.h"
+#include "Status.h"
+
+JsonString	StatusObject::toJsonString() const {
+	JsonString	jsonStringObj;
+	std::string jsonText = json_spirit::write_string(json_spirit::mValue(*this));
+	jsonStringObj.setJson(jsonText);
```

Comment:
While this isn't used a lot, when it is the string being copied could be very large so let's avoid the string copy.   This function body can be replaced with 

`return json_spirit::write_string(json_spirit::mValue(*this))`

if you define an rvalue constructor on JsonString that accepts std::string.